server/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java

```diff
@@ -83,7 +83,7 @@
 
     // see #3196
     public void testSuggestAcrossMultipleIndices() throws IOException {
-        createIndex("test");
+        prepareCreate("test").addMapping("type1", "text", "type=text").get();
```

Comment:
It might be nice to use `assertAcked` when creating indices, as we do in other places.

Reply:
Good idea, I'll do that.